velox/exec/tests/RowNumberTest.cpp

```diff
@@ -22,17 +22,32 @@ class RowNumberTest : public OperatorTestBase {};
 
 TEST_F(RowNumberTest, basic) {
   auto data = makeRowVector({
-      makeFlatVector<int64_t>({1, 2, 1, 2, 1, 2, 1}),
-      makeFlatVector<int64_t>({1, 2, 3, 4, 5, 6, 7}),
+      makeFlatVector<int64_t>({1, 2, 1, 2, 1, 2, 1}), // c0, paritioning key.
```

Comment:
Please remove these comments. The column names c0 and c1 are implicit and the usage of the partitioning and data depends on the individual tests.